AWS.md

```diff
@@ -78,3 +78,107 @@ The is no automatic installation of the proxy functions (yet).
 
 [invoke]: https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html
 
+## Getting started with `hedy` on AWS
+
+### 1. Create a Starter Project
+Create a simple JavaScript starter project. 
+
+src/index.js:
+
+```
+const { Response } = require('@adobe/fetch');
+
+/**
+ * This is the main function
+ * @param {Request} req Universal API Request
+ * @param {HEDYContext} context Universal API Context
+ * @returns {Response} a status response
+ */
+module.exports.main = async function main(req, context) {
+  const result = '{"result": "it works!"}';
+  return new Response(result, {
+    status: 200,
+  });
+}
+```
+
+package.json:
+```
+{
+  "name": "my-project",
+  "version": "1.0",
+  "private": true,
+  "main": "src/index.js",
+  "dependencies": {
+    "@adobe/fetch": "4.1.0"
+  }
+}
+```
+
+Build the project with `npm i`
+
+
+### 2. Configure your AWS account
+
+Its useful to install the AWS Commandline Client. See [Install or update the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
```

Comment:
It's actually a `Command Line Interface` see: 
<img width="563" alt="Screenshot 2023-10-04 at 14 09 25" src="https://github.com/adobe/helix-deploy/assets/22323322/374b3d2a-d54b-4c35-ae10-5b1b8e24481e">